### PR TITLE
passwall: dns2socks: socks server selectable

### PR DESCRIPTION
**Pull request by peter-tank**
Signed-off-by: peter-tank <30540412+peter-tank@users.noreply.github.com>

### CODE REVIEW

**Comment by Lienol** [comment]
If the test is ok and you think it's time to merge, leave a message.

**Comment by peter-tank** [comment]
@Lienol please review

**Comment by peter-tank** [comment]
hold, the new UI (socks_server selet list) has a problem on 19.07

File: lienol/luci-app-passwall/luasrc/model/cbi/passwall/global.lua [hunk 1]
```diff
@@ -208,8 +242,36 @@ o:value("global", translate("Global Proxy"))
...
+local tcp_node_num = tonumber(m:get("@global_other[0]", "tcp_node_num") or 1)
+for i = 1, tcp_node_num, 1 do
+    o:value("tcp" .. i, translatef("Same as the tcp %s node", i))
+end
+for k, v in pairs(nodes_table) do o:value(v.id, v.remarks) end
+
+o = s:option(Value, "port", translate("Listen Port"))
+o.datatype = "port"
+o.rmempty = false
+
```
Comment (Lienol) [inline lienol/luci-app-passwall/luasrc/model/cbi/passwall/global.lua:272]: get the socks status ajax request is not add.

File: lienol/luci-app-passwall/luasrc/model/cbi/passwall/other.lua [hunk 2]
```diff
@@ -181,34 +180,6 @@ uci:foreach(appname, "nodes", function(e)
...
-for i = 1, tcp_node_num, 1 do
-    o:value("tcp" .. i, translatef("Same as the tcp %s node", i))
-end
-for k, v in pairs(nodes_table) do o:value(v.id, v.remarks) end
-
-o = s:option(Value, "port", translate("Listen Port"))
-o.datatype = "port"
-o.rmempty = false
-
 m:append(Template(appname .. "/other/footer"))
```
Comment (Lienol) [inline lienol/luci-app-passwall/luasrc/model/cbi/passwall/other.lua]: move it to the global.lua

**Review by Lienol — CHANGES_REQUESTED** [review]

**Comment by peter-tank** [comment]
re-pushd, prevoisly tests fails on my wrong environments.

**Pull request merged**